### PR TITLE
IDP - Customer Tokens

### PR DESCRIPTION
**Pull request by nalancer08**
copilot:summary

### Type of change

- [ ] Chore
- [X] Feature
- [ ] Bug fix

### Linked issue

https://cybrid.atlassian.net/browse/CYB-1281

### Why

Replace bank token from SDK by customer token:

- DemoApp get bank token and customer token
- DemoApp get bank configuration
- SDK has new structure to init (need SDKConfig to run)

### Evidence

Captura de pantalla 2023-06-02 a la(s) 10 06 16

### CODE REVIEW

File: CybridSDKTestApp/CryptoAuthenticator.swift [hunk 1]
```diff
@@ -7,44 +7,47 @@
...
 
 class CryptoAuthenticator {
 
     private let session: URLSession
-    private let params = "banks:read banks:write accounts:read accounts:execute customers:read customers:write customers:execute prices:read quotes:execute trades:execute trades:read workflows:execute workflows:read external_bank_accounts:execute external_bank_accounts:read external_bank_accounts:write transfers:read transfers:execute"
-    private var clientID: String = ""
+    private let tokenScopes = "banks:read banks:write accounts:read accounts:execute customers:read customers:write customers:execute prices:read quotes:execute trades:execute trades:read workflows:execute workflows:read external_bank_accounts:execute external_bank_accounts:read external_bank_accounts:write transfers:read transfers:execute"
+    private let customerTokenScopes: Set<PostCustomerTokenIdpModel.ScopesIdpModel> = [
+        .customersRead, .customersWrite, .accountsRead, .accountsExecute, .pricesRead, .quotesRead, .quotesExecute, .tradesRead, .tradesExecute, .transfersRead, .transfersExecute, .externalBankAccountsRead, .externalBankAccountsWrite, .externalBankAccountsExecute, .workflowsRead, .workflowsExecute
+    ]
```
Comment (brent-cybrid) [inline CybridSDKTestApp/CryptoAuthenticator.swift]: You should put these scope lists in constants in a constants file

File: CybridSDKTests/Core/CDecimalTest.swift [hunk 2]
```diff
@@ -148,9 +148,9 @@ class CDecimalTest: XCTestCase {
         // -- When
         var decimal = CDecimal(valueToConvert)
         XCTAssertEqual(decimal.originalValue, valueToConvert)
-        XCTAssertEqual(decimal.intValue, BigInt(2))
-        XCTAssertEqual(decimal.decimalValue, "00")
-        XCTAssertEqual(decimal.newValue, "2.00")
+        XCTAssertEqual(decimal.intValue, decimalIntValue)
+        XCTAssertEqual(decimal.decimalValue, decimalDecimalValue)
+        XCTAssertEqual(decimal.newValue, decimalnewValue)
```
Comment (brent-cybrid) [inline CybridSDKTests/Core/CDecimalTest.swift]: `decimalNewValue`

Comment (nalancer08) [reply CybridSDKTests/Core/CDecimalTest.swift]: Done!

File: CybridSDKTestApp/CryptoAuthenticator.swift (hunk 1 above)
Comment (nalancer08) [reply CybridSDKTestApp/CryptoAuthenticator.swift]: Done!

**Review by brent-cybrid — APPROVED** [review]

**Pull request merged**